presto-hive/src/main/java/io/prestosql/plugin/hive/HiveMetadata.java

```diff
@@ -1738,6 +1740,41 @@ public void dropView(ConnectorSession session, SchemaTableName viewName)
         return tableNames.build();
     }
 
+    @Override
+    public Map<String, Object> getSchemaProperties(ConnectorSession session, CatalogSchemaName schemaName)
+    {
+        if (!filterSchema(schemaName.getSchemaName())) {
+            // "information_schema" and "sys" have no properties
+            return ImmutableMap.of();
+        }
+        Optional<Database> db = metastore.getDatabase(schemaName.getSchemaName());
+        if (db.isPresent()) {
+            Optional<String> location = db.get().getLocation();
+            // this is currently the only property for Hive
```

Comment:
I would you to use `io.prestosql.plugin.hive.HiveSchemaProperties#SCHEMA_PROPERTIES`. So whenever someone adds new schema property they don't need to do any changes here.

Reply:
Not sure what you mean. I would still need to add custom code here for each property to retrieved. For example the location is read from the Database object. Other properties have other ways to be retrieved.

Can you explain what you mean? Apologies if I'm missing something obvious.
Do you mean add a method to HiveSchemaProperties that is being passed a Database object? Or more like HiveTableProperties?

Reply:
I added method to HiveSchemaProperties. Lemme know what you think.